website/docs/r/continuous_compliance_notification.html.markdown

```diff
@@ -45,57 +45,89 @@ resource "dome9_continuous_compliance_notification" "test_notification" {
 
 The following arguments are supported:
 
-* `name` - (Required) The cloud account id.
-* `change_detection` - (Required) The account number.
+* `name` - (Required) The cloud account id in Dome9.
+* `description` - (Optional) Description of the notification.
+* `alerts_console` - (Optional) send  findings (also) to the Dome9 web app alerts console (Boolean); default is False.
+
+at least one of `scheduled_report` or `change_detection` must be included
+
+* `scheduled_report` - Scheduled email report notification  block:
+    * `email_sending_state` - send schedule report of findings by email; can be  "Enabled" or "Disabled.
+	
+	if `email_sending_state` is Enabled, the following must be included:
+	
+    * `schedule_data` -  Schedule details:
+        * `cron_expression` -  the schedule to issue the email report (in cron expression format)
+        * `type` - type of report; can be  "Detailed", "Summary", "FullCsv" or "FullCsvZip"
+        * `recipients` - comma-separated list of email recipients
```

Comment:
I think it's more clearly to add another tab for the following, then we can see that the `schedule_data` and the rest field are included to `email_sending_state`.